modules/elasticsearch/examples_test.go

```diff
@@ -0,0 +1,100 @@
+package elasticsearch_test
```

Comment:
Using Go examples for first time: I'm going to add them to all the modules, and the module generator, as they are true executable documentation that will land into the https://pkg.go.dev/github.com/testcontainers/testcontainers-go docs